doc/user/content/sql/create-cluster.md

```diff
@@ -74,15 +74,18 @@ Managed clusters with sources and sinks only support a replication factor of zer
 We plan to remove this restriction in a future version of Materialize.
 {{< /warning >}}
 
-## Syntax
+## Syntax to create managed clusters
+
 
 {{< diagram "create-managed-cluster.svg" >}}
 
 ### Cluster options
 
 {{% cluster-options %}}
 
-## Syntax
+We infer the value for `MANAGED` to be true unless you specify replicas manually.
```

Comment:
This line should be in the description field for the `MANAGED` row in the `{{% cluster options %}}` file. ([File link](https://github.com/antiguru/materialize/blob/create_cluster_rr/doc/user/layouts/shortcodes/cluster-options.html))